repo-validation/test-mit/LICENSE.md

```
@@ -0,0 +1,33 @@
+# MIT License
+
+Copyright (c) <no value> <no value>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: no

(MD033, no-inline-html)